accord-core/src/test/java/accord/topology/TopologyManagerTest.java

```diff
@@ -146,48 +146,33 @@ void syncCompleteFor()
         TopologyManager service = tracker();
 
         Assertions.assertFalse(service.getEpochStateUnsafe(2).syncComplete());
+        // shards to nodes: [[1, 2, 3], [4, 5, 6]]
+        // by syncing node 1/2 shard 1 has reached quorum, but not shard 2
         service.onEpochSyncComplete(id(1), 2);
         service.onEpochSyncComplete(id(2), 2);
         Assertions.assertFalse(service.getEpochStateUnsafe(2).syncComplete());
         Assertions.assertTrue(service.getEpochStateUnsafe(2).syncCompleteFor(keys(150).toParticipants()));
         Assertions.assertFalse(service.getEpochStateUnsafe(2).syncCompleteFor(keys(250).toParticipants()));
     }
 
-    /**
-     * Epochs should only report being synced if every preceding epoch is also reporting synced
-     */
     @Test
-    void existingEpochPendingSync()
```

Comment:
deleted the test as its no longer relevant... making sure that a new epoch doesn't resolve the past is no longer true... new epochs resolve the previous ones!